Makefile

```diff
@@ -146,6 +173,129 @@ e2e-helm-install:
 	cd .staging/helm && tar -xvf helmbin.tar.gz
 	./.staging/helm/linux-amd64/helm version --client
 
+e2e-notaryv2-setup:
+	rm -rf .staging/notaryv2
+	mkdir -p .staging/notaryv2
+	cd .staging/notaryv2 && git clone https://github.com/notaryproject/notation
+	cd .staging/notaryv2/notation && make build
+
+	echo 'FROM alpine\nCMD ["echo", "notaryv2 signed image"]' > .staging/notaryv2/Dockerfile
+	docker build -t ${LOCAL_TEST_REGISTRY}/notation:signed .staging/notaryv2
+	docker push ${LOCAL_TEST_REGISTRY}/notation:signed
+
+	docker pull ${LOCAL_UNSIGNED_IMAGE}
+	docker image tag ${LOCAL_UNSIGNED_IMAGE} ${LOCAL_TEST_REGISTRY}/notation:unsigned
+	docker push ${LOCAL_TEST_REGISTRY}/notation:unsigned
+
+	rm -rf ~/.config/notation
+	.staging/notaryv2/notation/bin/notation cert generate-test --default "ratify-bats-test"
```

Comment:
wonder if we need to do the same thing to local images? https://github.com/deislabs/ratify/issues/504

Reply:
nvm, it's done in `e2e-licensechecker-setup`